content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -290,6 +290,11 @@ You can remove Pods from a ReplicaSet by changing their labels. This technique m
 from service for debugging, data recovery, etc. Pods that are removed in this way will be replaced automatically (
 assuming that the number of replicas is not also changed).
 
+Change the label name in the POD YAML definition:
+```shell
+kubectl edit POD <POD NAME>
```

Comment:
Should the first `POD` be `pod`? Also, in line 198 of this file, `kubectl get Pods` should be `kubectl get pods`